backend/src/ast/converters/python/operators.ts

```diff
@@ -202,3 +202,10 @@ export const generatorExpressionOperator = "generator-expression";
  *           ^^^^^^^^^^^^^
  */
 export const notOperator = "not";
+
+/**
+ * Matches a double-star kwarg in function calls
+ *   func(**kwargs)
+ *        ^^^^^^^^
+ */
+export const doubleStarKwArgOperator = "double-star-kwarg";
```

Comment:
minor pet peeve, but it's typically `kwargs` with an `s`, as the operator would make no sense unless there's multiple keyword arguments :-)